client_local_api/api_test_utils.go

```diff
@@ -0,0 +1,30 @@
+package client_local_api
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+//TODO move this to random port for testing
+const serverBindAddress = ":8000"
+
+func startServer() {
+	go Bootstrap(serverBindAddress)
+}
+
+func doGet(t *testing.T, path string) *http.Response {
+
+	resp, err := http.Get("http://localhost" + serverBindAddress + "/" + path)
```

Comment:
I would use 127.0.0.1 instead of localhost since some environments may not have localhost -> 127.0.0.1 mapping.

Reply:
This is just a test - server is started on localhost, and client acess it during part of testing.